lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -100,7 +100,7 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     "Green-B" => ["Boston College", "Government Center"],
     "Green-C" => ["Cleveland Circle", "Government Center"],
     "Green-D" => ["Riverside", "North Station"],
```

Comment:
Ope! Similarly, this should be "[Union Sq](https://www.mbta.com/schedules/Green-D/line?schedule_direction%5Bdirection_id%5D=0)"

Reply:
Good catch! Thanks!